### DISCUSSION TITLE
Step 2 of Github Copilot is not generating any code for me as per Read me

### DISCUSSION THREAD

**Discussion by prmangen** [skills/.github#282] [Code with Copilot]
![image](https://github.com/skills/.github/assets/47916604/0e657b64-be67-4dda-b890-3ee78737a170)

Reality 

Documentation says the below should be the generation of the code. 

![image](https://github.com/skills/.github/assets/47916604/62af6225-2f5a-455f-8622-977b10aa4edc)

**Comment by sn2979** [comment]
Same thing happened to me

**Comment by Herman-Motcheyo** [comment]
same thing happened to me

**Comment by Jusmic** [comment]
likewise for me

**Comment by AswinSuku** [comment]
I think we need to take the copilot trial package (($10/month or $100/year package)  for auto generation of code  : (

**Comment by slingin-simmo** [comment]
I have taken the trial package and put my payment details, and it still isn't working for me either.

**Comment by racheltienng** [comment]
After putting the codes to commit and push in the TERMINAL next to the Debug Console on the lower side:
![image](https://github.com/skills/.github/assets/65115880/af8d81f5-f6a4-46a5-af9d-b68205efaacd)

Then you can workaround by:
Go to your main GitHub-CoPilot page
Click folder Steps
Click on 3-Copilot-Hub

![Screenshot 2023-07-18 at 16 23 13](https://github.com/skills/.github/assets/65115880/52658421-6517-417b-a42e-8f901e98178b)

I agree that the instructions in the activity in Step 2 create confusion. I had to search around 30mins where the Terminal was. And after refreshing I couldn't go to Step 3. Has to workaround and find it myself.

**Comment by prmangen** [comment]
Sorry not sure what is missing here.

I have followed your revised instructions however I still do not see the code suggestions.   I added the braces to type something but no suggestions
[cid:c642f550-5ea0-4817-9d09-ef80b9da1296]
________________________________

**Reply by racheltienng** [reply]
I'm having trouble seeing your screenshot. Tried to push the code to your repository in the terminal and not where you put the function.
And then navigate back to your github homepage, then GitHub-Copilot, then the Folder Steps to see the Steps and click on Step 3.

**Comment by sinsukehlab** [comment]
Try pressing "[Ctrl] + [Enter]".
